app/models/question.rb

```diff
@@ -3,4 +3,9 @@ class Question < ActiveRecord::Base
   has_and_belongs_to_many :templates
   validates :content,  presence: true, uniqueness: { case_sensitive: false }
   validates :topic,    presence: true
+  validates :level, 	 presence: true
+
+  def possible_levels
+    ['beginner', 'good', 'strong']
+  end
```

Comment:
It is better to use class methods for such things.
And constnat is even better than class method.